core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1254,15 +1261,37 @@ public function getArticleAlias(Contao\DataContainer $dc)
 		return $arrAlias;
 	}
 
+	/**
+	 * Saves the article alias
```

Comment:
This does not seem to match what the method does, does it?

Reply:
Changed in b1eeebb30a3fa26f5bc1c042e91ed6889585acb1